### DISCUSSION TITLE
欢迎来到element3技术讨论区

### DISCUSSION THREAD

**Discussion by shengxinjing** [hug-sun/element3#449] [General]
## 👋 欢迎
为了更好的进行讨论，说明下本讨论区规则

1. 只讨论和「element3」相关的内容
2. 不要发布广告，维护良好的社区环境，谢谢。

如果你喜欢更直接的讨论方式，可以加入社区钉钉群
![Jietu20201209-221830@2x](https://user-images.githubusercontent.com/1905176/101641250-7d6b6000-3a6c-11eb-891a-bf2c42c6f25f.jpg)

**Comment by cwandev** [comment]
请问element3是怎么做响应式的？
